lib/elixir/lib/code.ex

```diff
@@ -594,10 +594,10 @@ defmodule Code do
   ## Examples
 
       # Get the documentation for the first function listed
-      iex> [fun | _] = Code.get_docs(Atom, :docs) |> Enum.sort()
+      iex> [fun | _] = Code.get_docs(Atom, :docs) |> Enum.sort() |> tl
```

Comment:
We are saying "Get the documentation for the first function listed".
We should rather change expected result in doctest.